code/game/objects/items/robot/cyborg_gripper.dm

```diff
@@ -0,0 +1,223 @@
+/*   CONTENTS:
+0. generic define gripper
+1. UNIVERSAL GRIPPER
+2. MEDICAL GRIPPER
+3. SERVICE GRIPPER
+4. ENGINEERING GRIPPER
+*/
+
+// Generic gripper. This should never appear anywhere.
+/obj/item/gripper
+	name = "generic gripper item"
+	desc = "If you can see this, make an issue report to Github. Something has gone wrong!"
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gripper"
+	actions_types = list(/datum/action/item_action/drop_gripped_item)
+	/// Set to TRUE to allow interaction with light fixtures and cell-containing machinery.
+	var/engineering_machine_interaction = FALSE
+	/// Set to TRUE to allow the gripper to shake people awake/help them up.
+	var/can_help_up = FALSE
+	/// Defines what items the gripper can carry.
+	var/list/can_hold = list()
+	/// The item currently being held.
+	var/obj/item/gripped_item
+
+/obj/item/gripper/examine_more(mob/user)
+	. = ..()
+	. += "Cyborg grippers are well-developed, and despite some anatomical differences that manifest in some models, \
+	they can be used just as effectively as a regular hand with enough practice. \
+	Companies like Nanotrasen use software to limit the items that a cyborg can manipulate to a specific pre-defined list, as part of their multi-layered \
+	protections to try and eliminate the chance of a hypothetical synthetic uprising, not wishing to see a repeat of the IPC uprising in 2525."
+
+
+/obj/item/gripper/Initialize(mapload)
+	. = ..()
+	can_hold = typecacheof(can_hold)
+
+/obj/item/gripper/ui_action_click(mob/user)
+	drop_gripped_item()
+
+/obj/item/gripper/proc/drop_gripped_item(silent = FALSE)
+	if(!gripped_item)
+		return
```

Comment:
Can you pass `user` in the arguments here?

Reply:
Done.